app/Http/Controllers/Api/Instructor/ChapterController.php

```diff
@@ -172,21 +172,18 @@ public function patchStatus(PatchStatusRequest $request, UpdateChapterStatusServ
      */
     public function bulkDelete(BulkDeleteRequest $request, BulkDeleteChapterService $bulkDeleteChapterService): JsonResponse
     {
-        // 認証ユーザー情報取得
-        $instructorId = Auth::guard('instructor')->user()->id;
-
         $chapterIds = $request->input('chapters', []);
         $courseId = $request->input('course_id');
 
         try {
             $chapters = Chapter::with(['course', 'lessons'])->whereIn('id', $chapterIds)->get();
-            $chapters->each(function (Chapter $chapter) use ($instructorId, $courseId) {
-                if ((int) $instructorId !== $chapter->course->instructor_id) {
-                    // チャプターに紐づく講師でない場合は許可しない
-                    throw new AuthorizationException('Forbidden, invalid instructor_id.');
-                }
+            $chapters->each(function (Chapter $chapter) use ($courseId) {
+
+                // 認可処理 policy使用
+                $this->authorizeResource('bulkDelete', [Chapter::class, $chapter]);
+
```

Comment:
段階的に認可処理を集約していくので、次のようなコードになる想定でした！

```suggestion
            $this->authorize('bulkDelete', [Chapter::class, $chapters]);
            
            $chapters->each(function (Chapter $chapter) use ($courseId) {
                if ((int) $courseId !== $chapter->course_id) {
                    // 指定した講座に属するチャプターでなければエラー応答
                    throw new AuthorizationException('Forbidden, invalid course_id.');
                }
            });
```